app/sdil/controllers/ReturnJourney.scala

```diff
@@ -19,20 +19,24 @@ import java.time.LocalDateTime
 
 import scala.concurrent._
 import cats.implicits._
+import javax.inject.Inject
 import ltbs.play.scaffold.GdsComponents._
 import ltbs.play.scaffold.SdilComponents._
 import play.api.i18n.Messages
 import play.api.libs.json.{Format, JsValue}
+import play.api.mvc.MessagesControllerComponents
+import sdil.config.AppConfig
 import sdil.connectors.SoftDrinksIndustryLevyConnector
 import sdil.models._
 import sdil.models.retrieved.RetrievedSubscription
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.uniform.playutil.ExtraMessages
 import uk.gov.hmrc.uniform.webmonad._
+import views.uniform.Uniform
 
-trait ReturnJourney extends SdilWMController {
-
-  implicit val ec: ExecutionContext
+class ReturnJourney @Inject()(uniformHelpers: Uniform, mcc: MessagesControllerComponents, config: AppConfig)(
```

Comment:
This is only used to extends the other controllers, so I think it was better off being a trait

Reply:
Modified 2 classes back to trait